users/player-info-forwarding.rst

```diff
@@ -17,7 +17,22 @@ forwarding player information to your servers:
 Configuring modern forwarding
 -----------------------------
 
-Currently, no server implementations support Velocity's modern forwarding.
+Currently, only builds 337 and above of Paper 1.13.1+ support Velocity's modern forwarding.
```

Comment:
https://papermc.io/ci/job/Paper-1.13/377/ is the build that introduced Velocity support.